cmake/evt-core_compiler.cmake

```diff
@@ -32,6 +33,10 @@ else()
 endif()
 message(STATUS "${TARGET_DEV} targeted")
 
+option (FORCE_COLORED_OUTPUT "Always produce ANSI-colored output (GNU/Clang only)." OFF)
+if (FORCE_COLORED_OUTPUT)
```

Comment:
You put spaces after all your keywords, but no other lines in this file do that. I would prefer that you conform.

Reply:
work habit, it is invading.